SamplifyAPIClient/Query.cs

```diff
@@ -82,14 +82,16 @@ public QueryOptions(uint offset, uint limit)
             this.Limit = limit;
         }
 
-        public void AddFilter(string field, object value)
+        public QueryOptions AddFilter(string field, object value)
```

Comment:
why is this return type changed?

Reply:
builder design pattern. kind of subjective.

Reply:
we can chain the AddFilter method